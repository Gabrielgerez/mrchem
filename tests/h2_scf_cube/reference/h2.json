{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            0.0,
            0.0,
            0.672828
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            0.0,
            0.0,
            -0.672828
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_per_node": -1,
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000,
      "use_omp_num_threads": false
    },
    "mra": {
      "basis_order": 7,
      "basis_type": "legendre",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "h2",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "rotate": true,
        "screen": 12.0,
        "type": "cube",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "environment": "None",
        "external_field": "None",
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 3,
        "localize": false,
        "max_iter": 5,
        "method": "Hartree-Fock",
        "orbital_thrs": 0.01,
        "relativity": "None",
        "rotation": 0,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -1.0790011801500128e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.00012092406391447963,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            8.547430783368667e-05,
            8.547430783737533e-05,
            -3.30373026546085e-06
          ],
          "vector_el": [
            8.547430717730367e-05,
            8.547430717487773e-05,
            -3.3037302634624487e-06
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.672828
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.672828
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.6040968973290005
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.208193794658001
      },
      "scf_energy": {
        "E_ee": 1.3367894596221979,
        "E_eext": 0.0,
        "E_el": -1.8765910088977398,
        "E_en": -3.7001047171356194,
        "E_kin": 1.155116493998143,
        "E_next": 0.0,
        "E_nn": 0.7431319742935788,
        "E_nuc": 0.7431319742935788,
        "E_tot": -1.133459034604161,
        "E_x": -0.6683922453824612,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.353822648870343,
        "E_eext": 0.0,
        "E_el": -1.8288574942268974,
        "E_en": -3.7378031794343007,
        "E_kin": 1.2333096343690924,
        "E_next": 0.0,
        "E_nn": 0.7431319742935788,
        "E_nuc": 0.7431319742935788,
        "E_tot": -1.0857255199333187,
        "E_x": -0.6781865980320322,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 1.334102156803025,
              "E_eext": 0.0,
              "E_el": -1.8760154161322915,
              "E_en": -3.6924635391293887,
              "E_kin": 1.1493944585501088,
              "E_next": 0.0,
              "E_nn": 0.7431319742935788,
              "E_nuc": 0.7431319742935788,
              "E_tot": -1.1328834418387128,
              "E_x": -0.6670484923560364,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1328834418387128,
            "energy_update": 0.04715792190539414,
            "mo_residual": 0.048072947749071754,
            "wall_time": 1.35335435
          },
          {
            "energy_terms": {
              "E_ee": 1.3367894596221979,
              "E_eext": 0.0,
              "E_el": -1.8765910088977398,
              "E_en": -3.7001047171356194,
              "E_kin": 1.155116493998143,
              "E_next": 0.0,
              "E_nn": 0.7431319742935788,
              "E_nuc": 0.7431319742935788,
              "E_tot": -1.133459034604161,
              "E_x": -0.6683922453824612,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.133459034604161,
            "energy_update": 0.000575592765448274,
            "mo_residual": 0.009349602054305949,
            "wall_time": 1.338637456
          }
        ],
        "wall_time": 2.6933379
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
