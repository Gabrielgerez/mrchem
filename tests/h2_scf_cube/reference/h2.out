                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         text                                     ***
***         Git commit hash    20e53b0ebebbbed503cb-dirty               ***
***         Git commit author  gitpeterwind                             ***
***         Git commit date    Thu Feb 20 16:31:25 2025 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : merge
 Git commit hash       : d82d0d16fbefc341d12f
 Git commit author     : gitpeterwind
 Git commit date       : Thu Feb 20 16:02:13 2025 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 7
 polynomial type       : Legendre
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.672828
    1       H            :        0.000000        0.000000       -0.672828
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project cube file molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : cube_vectors/CUBE_p_vector.json
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.990573017548e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.233309634369
 E-N energy              :            (au)                 -3.737803179434
 Coulomb energy          :            (au)                  1.353822648870
 Exchange energy         :            (au)                 -0.678186598032
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.743131974294
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.828857494227
 Nuclear energy          :            (au)                  0.743131974294
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.085725519933e+00
                         :      (kcal/mol)             -6.813030499902e+02
                         :        (kJ/mol)             -2.850571961159e+03
                         :            (eV)             -2.954409648003e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.577699400727
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.155398801454
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : Hartree-Fock
 Relativity              : None
 Environment             : None
 External fields         : None
 Checkpointing           : Off
 Max iterations          : 5
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00          -1.085725519933         -1.085726e+00
    1          4.807295e-02          -1.132883441839         -4.715792e-02
    2          9.349602e-03          -1.133459034604         -5.755928e-04
---------------------------------------------------------------------------
                      SCF converged in 2 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.672828
    1       H            :        0.000000        0.000000       -0.672828
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.155116493998
 E-N energy              :            (au)                 -3.700104717136
 Coulomb energy          :            (au)                  1.336789459622
 Exchange energy         :            (au)                 -0.668392245382
 X-C energy              :            (au)                  0.000000000000
 N-N energy              :            (au)                  0.743131974294
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.876591008898
 Nuclear energy          :            (au)                  0.743131974294
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133459034604e+00
                         :      (kcal/mol)             -7.112562826765e+02
                         :        (kJ/mol)             -2.975896286718e+03
                         :            (eV)             -3.084299158462e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.604096897329
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.208193794658
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000085        0.000085       -0.000003
 Magnitude               :            (au)                        0.000121
                         :         (Debye)                        0.000307
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000085        0.000085       -0.000003
 Magnitude               :            (au)                        0.000121
                         :         (Debye)                        0.000307
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  3s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
